component/prometheus/operator/podmonitors/config_gen/config_gen.go

```diff
@@ -0,0 +1,164 @@
+package config_gen
```

Comment:
It's typically unidiomatic to use underscores in package names, unless it's representing  a named entity that already uses underscores (like an integration referencing an exporter like node_exporter or a component like kubernetes_events).